itu-minitwit-golang/internal/service/logout_handler.go

```diff
@@ -0,0 +1,13 @@
+package service
+
+import (
+	"github.com/gin-contrib/sessions"
+	"github.com/gin-gonic/gin"
+)
+
+
+func LogOutHandler(c *gin.Context) {
```

Comment:
You dont actually logout the user, you just get the current session and add a flash